### PR TITLE
Delay between retries

### PR DESCRIPTION
**Pull request by jtobard**
**Time delay** 

Added in job edit/create page:
captura de pantalla 2017-06-21 a la s 10 40 20
Uses the same format for timeout option and accept option value as input too.

ref. issue: #2083

### CODE REVIEW

**Comment by ahonor** [comment]
Update the job format reference docs (yaml/xml) and also update the manual page for Jobs.

**Comment by jtobard** [comment]
Updated doc and new format for xml:
`<retry delay='1h1m1s'>2</retry>`

File: rundeckapp/grails-app/domain/rundeck/ScheduledExecution.groovy [hunk 1]
```diff
@@ -328,8 +328,12 @@ class ScheduledExecution extends ExecutionContext {
             se.uuid = data.uuid
         }
         se.timeout = data.timeout?data.timeout.toString():null
-        se.retry = data.retry?data.retry.toString():null
-        se.retryDelay = data.retryDelay?data.retryDelay.toString():null
+        if(data.retry instanceof Map){
+            se.retry = data.retry.retry?data.retry.retry.toString():null
```
Comment (gschueler) [inline rundeckapp/grails-app/domain/rundeck/ScheduledExecution.groovy]: just a style comment: but you can do this: 

~~~{.groovy}
se.retry = data.retry.retry?.toString() ?: null
~~~

Comment (gschueler) [inline rundeckapp/grails-app/domain/rundeck/ScheduledExecution.groovy]: actually even this `data.retry.retry?.toString()` (same thing)

**Pull request merged**